core/modules/system/system.menu.inc

```diff
@@ -157,9 +161,15 @@ function system_menu_block_build(array $config) {
     'style' => 'tree',
     'level' => 1,
     'depth' => 3,
-    'expand_all' => FALSE,
+    'expand_all' => TRUE,
   );
 
+  if ($config['style'] == 'tree') {
+    $config += array(
+      'expand_all' => FALSE,
+    );
```

Comment:
This `+=` operator will never do anything, as the `+=` operator above will have already added the `'expand_all' => TRUE,` item.

Reply:
bah.